### ISSUE TITLE
 module's bundleconfig.json  is not work

### ISSUE THREAD

**Issue by daxiongok** [simplcommerce/SimplCommerce#942]
hi dear author，Im builded the solution，but module's bundleconfig.json  is not generate min-file ,What I need to configure？

**Comment by TATARPRO** [comment]
install the [BundleMinfier ](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.BundlerMinifier)vs extension

**Comment by daxiongok** [comment]
> install the [BundleMinfier ](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.BundlerMinifier)vs extension

yeah :),thanks,there's new problem that  how to run module‘s min.js in the debugger mode . how to  configure module js path?
